src/decode.rs

```diff
@@ -909,43 +907,28 @@ impl Buffer {
         }
     }
 
-    fn reconstruct_high(&mut self) -> u8 {
+    fn fill_fast(&mut self) -> u8 {
```

Comment:
The name `fill_fast` does not do a better job of explaining that this is the special case for lzw reconstruction. It shouldn't be a problem to be verbose here.

Reply:
Plus a comment while you're at it :pleading_face:

Reply:
changed to `fill_cscsc` which is not a better name but should draw attention to the comment :*